### PR TITLE
fix(action): Prevent disabled buttons from being clicked or tabbed into

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** None

## Summary

fix(action): Prevent disabled buttons from being clicked or tabbed into

### CODE REVIEW

**Comment by asangma** [comment]
Shouldn't browsers default to this behavior with buttons that have a `disabled` attribute?

**Comment by driskull** [comment]
> Shouldn't browsers default to this behavior with buttons elements that have a disabled attribute?

Yes, but when users set an onClick on the element it isn't set on the `button`. So this is needed.

**Comment by asangma** [comment]
Ah...got it.

File: src/components/calcite-action/calcite-action.tsx [hunk 1]
```diff
@@ -182,6 +182,7 @@ export class CalciteAction {
     return (
       <Host>
         <button
+          tabIndex={disabled ? -1 : null}
```
Comment (jcfranco) [inline src/components/calcite-action/calcite-action.tsx:185]: `disabled` should take care of the tab index, no?

File: src/components/calcite-action/calcite-action.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@
   background-color: var(--calcite-app-background-clear);
 }
 
+:host([disabled]) {
```
Comment (jcfranco) [inline src/components/calcite-action/calcite-action.scss:7]: Can you add a test for this?

**Pull request merged**